tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -480,6 +480,102 @@ public function test_payment_fields_outputs_fields() {
+		string $adaptive_pricing,
+		bool $expect_selector
+	): void {
+		$gateway = $this->getMockBuilder( WC_Stripe_UPE_Payment_Gateway::class )
+			->setConstructorArgs( [] )
+			->onlyMethods( [ 'get_return_url', 'is_valid_optimized_checkout_page' ] )
+			->getMock();
+		$gateway->method( 'get_return_url' )->willReturn( self::MOCK_RETURN_URL );
+		$gateway->method( 'is_valid_optimized_checkout_page' )->willReturn( $valid_oc_page );
+		$gateway->oc_enabled = $oc_enabled;
+
+		$original_stripe_settings         = WC_Stripe_Helper::get_stripe_settings();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'test_payment_fields_renders_currency_selector_conditionally'. (undefined)

(StaticAccess)